package.json

```diff
@@ -101,14 +98,11 @@
     "nodemon": "^1.18.3",
     "pixelmatch": "^4.0.2",
     "pm2": "^2.2.3",
-    "postcss-loader": "^0.9.1",
     "precommit-hook": "^3.0.0",
     "puppeteer": "^1.3.0",
     "redux-mock-store": "^1.0.2",
     "rimraf": "^2.5.2",
-    "sass-loader": "^7.0.3",
     "strip-loader": "^0.1.0",
-    "style-loader": "^0.13.1",
     "url-loader": "^0.5.7",
```

Comment:
Could we also remove 
`strip-loader`,
`url-loader`,
`extract-text-webpack-plugin`,
`file-loader`,
`json-loader`,
`node-sass`,
`classnames`
?

Reply:
By the way, there are many packages to remove after we upgrade article component to v2.
If available, we could create another PR to remove them from package.json.